tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot

```diff
@@ -35,7 +38,12 @@ End Web Billing Metrics Test
   SeleniumLibrary.Close All Browsers
 
 Skip Test If Previous CPU Usage Is Not Zero
```

Comment:
is this keyword needed? it seems to be not used

Reply:
Removed!